kv/kv.go

```diff
@@ -0,0 +1,155 @@
+package kv
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/jordan-rash/wasmcloud-go/models"
+	"github.com/nats-io/nats.go"
+	core "github.com/wasmcloud/interfaces/core/tinygo"
+)
+
+const (
+	LINKDEF_PREFIX = "LINKDEF_"
+	CLAIMS_PREFIX  = "CLAIMS_"
+	BUCKET_PREFIX  = "LATTICEDATA_"
+)
+
+var (
+	DEFAULT_LEAFNOTE_OPTIONS nats.JSOpt = nats.PublishAsyncMaxPending(256)
```

Comment:
Nit: Typo in `LEAFNOTE`?

Reply:
Nope! This is where I keep my notes... where do you keep yours?